src/resolutions/lesson-02/8-ts-inference.ts

```diff
@@ -0,0 +1,40 @@
+import { Equal, Expect } from "../../../helper";
+/*
+9. don't tell me you forgot it already? TS inference 
+(look for what the test 'line 193' is expecting) and type the function correctly using generics
+if we remove @ts-expect-error from line 199 and 204 it should give an error if the solucion is correct
+*/
+
+type AllowedType<T> = T extends string | number ? T : never;
```

Comment:
`: never` no debe ir, siempre queremos un tipo asociado a nuestros métodos, así al momento de no pasar el tipo correcto nuestro IDE nos puede decir cual es el correcto.

Reply:
Thx, addressed in next commit.